worlds/ladx/Options.py

```diff
@@ -0,0 +1,384 @@
+import os.path
+import typing
+import logging
+from Options import Choice, Option, Toggle, DefaultOnToggle, Range, FreeText
+from collections import defaultdict
+
+DefaultOffToggle = Toggle
+
+logger = logging.getLogger("Link's Awakening Logger")
+
+
+class LADXROption:
+    def to_ladxr_option(self, all_options):
+        if not self.ladxr_name:
+            return None, None
+        
+        return (self.ladxr_name, self.name_lookup[self.value].replace("_", ""))
+
+
+class Logic(Choice, LADXROption):
+    """Affects where items are allowed to be placed.
+    [Casual] Same as normal, except that a few more complex options are removed, like removing bushes with powder and killing enemies with powder or bombs.
+    [Normal] Playable without using any tricks or glitches. Can require knowledge from a vanilla playthrough, such as how to open Color Dungeon.
+    [Hard] More advanced techniques may be required, but glitches are not. Examples include tricky jumps, killing enemies with only pots.
+    [Glitched] Advanced glitches and techniques may be required, but extremely difficult or tedious tricks are not required. Examples include Bomb Triggers, Super Jumps and Jesus Jumps.
+    [Hell] Obscure knowledge and hard techniques may be required. Examples include featherless jumping with boots and/or hookshot, sequential pit buffers and unclipped superjumps. Things in here can be extremely hard to do or very time consuming."""
+    display_name = "Logic"
+    ladxr_name = "logic"
+    # option_casual = 0
+    option_normal = 1
+    option_hard = 2
+    option_glitched = 3
+    option_hell = 4
+    
+    default = option_normal
+
+class TradeQuest(DefaultOffToggle, LADXROption):
+    """
+    On - adds the trade items to the pool (the trade locations will always be local items)
+    Off - (default) doesn't add them
+    """
+    ladxr_name = "tradequest"
+
+
+#            Setting('forwardfactor', 'Main', 'F', 'Forward Factor', default=0.0,

```

Comment:
looks like leftover stuff that could be removed? I found a bunch of single line old code comments as well, but this one is a bit big